homeassistant/components/mqtt/camera.py

```diff
@@ -108,6 +113,13 @@ class MqttCamera(MqttEntity, Camera):
     def __init__(self, hass, config, config_entry, discovery_data):
         """Initialize the MQTT Camera."""
         self._last_image = None
+        # Using CONF_ENCODING to set b64 encoding for images is deprecated as of Home Assistant 2022.9, use CONF_IMAGE_ENCODING instead
+        if config[CONF_ENCODING] == "b64":
+            config[CONF_IMAGE_ENCODING] = "b64"
+            config[CONF_ENCODING] = DEFAULT_ENCODING
+            _LOGGER.warning(
+                "Using the `encoding` parameter to set image encoding has been deprecated, use `image_encoding` instead"
+            )
```

Comment:
Do this as a part of validation, and use the `repairs` API instead of logging a warning.

Reply:
Can you have a look?

Reply:
Have this now:
![afbeelding](https://user-images.githubusercontent.com/7188918/182876056-8c5402b1-3220-448d-b46e-d0ac6d732933.png)
Only uploaded and English translation.

Reply:
Reverted the changes using the repairs API for this PR.